site/docs/integrations/rollup.md

```diff
@@ -90,3 +90,13 @@ Each integration will set a default value based on the configuration options pas
 
 esbuild is used internally to compile `.css.ts` files before evaluating them to extract styles. You can pass additional options here to customize that process.
 Accepts a subset of esbuild build options (`plugins`, `external`, `define`, `loader`, `tsconfig` and `conditions`). See the [build API](https://esbuild.github.io/api/#build-api) documentation.
+
+### extract
+
+Extract all generated `.css` into one bundle. Specify filename relative to `output.dir`.
+
+```ts
+vanillaExtractPlugin({
+  extract: 'app.css'
+});
+```
```

Comment:
Looks like the config options need to be updated to reflect the new config. Additionally, should the fact that `output.assetFileNames` is respected also be mentioned?

Reply:
Oh good catch. Added that note as well!